src/csv/load_csv.h

```diff
@@ -68,6 +69,11 @@ typedef struct s_csv_context {
   size_t num_columns;                            // Number of columns read from CSV (headers).
 } csv_context_t;
 
+#  define CSV_CONTEXT()                                                                            \
+    (csv_context_t) {                                                                              \
+      .filepath = NULL, .num_column_defs = 0, .row_cap = 0, .num_rows = 0, .num_columns = 0       \
+    }
```

Comment:
See the solution that I found and used in io_layer_distribution.h.
```
static const struct layers_T {
  int number_of_layers;
  double *
      normalized_target_volumes; // Percentage of water volume per layer, discharge will be transferred proportionally
} layers_default = {.number_of_layers = 0, .normalized_target_volumes = NULL};

typedef struct layers_T layers;
```
With this technique you can prevent using a macro for creating the default